### ISSUE TITLE
ticket lifetime

### ISSUE THREAD

**Issue by ms-s** [emu-wg/draft-ietf-emu-eap-tls13#23]
full handshake.  It is left up to the EAP-TLS peer whether to use
    resumption, but it is RECOMMENDED that the EAP-TLS server accept
    resumption as long as the ticket is valid.  However, the EAP-TLS

But the EAP-TLS server sets how long the ticket is valid
Should we give some guidance on how long that should be (noting that it
must say within the RFC 8446-mandated 7 day maximum).

**Comment by emanjon** [comment]
Not sure EAP-TLS should give more TLS recommendations here.

I think 
"but it is RECOMMENDED that the EAP-TLS server accept resumption as long as the ticket is valid."

was added on request by Alan. TLS just says that the Server may or may not accept its own valid ticket. If I remember correctly.

**Comment by ms-s** [comment]
Perhaps it is sufficient to rephrase as "It is left up to the EAP-TLS peer whether to use resumption, but it is RECOMMENDED that the EAP-TLS server accept resumption if the ticket that it issued is still valid. Note that RFC8446 limits the ticket lifetime to a maximum of  604800 seconds (7 days) and EAP-TLS servers MUST respect this upper limit when issuing tickets."

**Comment by emanjon** [comment]
I made commit e9d528d.

Adding more text on ticket lifetime as raised by Ben and discussed in the issue tracker. Added the text suggested by Mohit. Also added several sentences describing that a ticket contains an identity and a PSK lifetime. Added info a key share in the full authentication so it does not show up only in resumption. Added text that key share and psk mode affect the security properties. And added text that EAP-TLS 1.3 provides forward secrecy.

With that I think we can close the issue
